src/main/resources/site/content-types/statistics/statistics.xml

```diff
@@ -1,145 +1,200 @@
-        </field-set>
-
-        <field-set>
-            <label>Om Statistikken</label>
-            <items>
-                <input name="aboutTheStatistics" type="ContentSelector">
-                    <label>Om Statistikken</label>
-                    <occurrences minimum="0" maximum="1"/>
-                    <config>
-                        <allowType>omStatistikken</allowType>
-                        <description>Legg til Om Statistikken</description>
-                    </config>
-                </input>
-            </items>
-        </field-set>
-
-        <field-set>
-            <label>Nøkkeltall</label>
-            <items>
-                <input name="statisticsKeyFigure" type="ContentSelector">
-                    <label>Nøkkeltall</label>
-                    <occurrences minimum="0" maximum="1"/>
-                    <config>
-                        <allowType>keyFigure</allowType>
-                        <description>Legg til nøkkeltall</description>
-                    </config>
-                </input>
-            </items>
-        </field-set>
-
-        <field-set>
-            <label>Statbank lenkeliste</label>
-            <items>
-                <item-set name="statbankLinkItemSet">
-                    <label>Statbank lenker</label>
-                    <occurrences minimum="0" maximum="5"/>
-                    <items>
-                        <input type="TextLine" name="urlText">
-                            <label>Lenketekst</label>
-                            <occurrences minimum="1" maximum="1" />
-                        </input>
-                        <input type="TextLine" name="url">
-                            <label>URL</label>
-                            <occurrences minimum="1" maximum="1" />
-                        </input>
-                    </items>
-                </item-set>
-                <input type="TextLine" name="linkNumber">
-                    <label>Antall tabeller i statbank</label>
-                    <occurrences minimum="0" maximum="1" />
-                </input>
-            </items>
-        </field-set>
-
-        <mixin name="relatedStatistics"/>
-
-        <mixin name="relatedExternalLinks" />
-
-        <mixin name="relatedArticles" />
-
-        <mixin name="relatedFactPage" />
-
-        <mixin name="contacts" />
-
-        <field-set>
-            <label>Stikkord</label>
-            <items>
-                <input name="keywords" type="TextArea">
-                    <label>Stikkord</label>
-                    <occurrences minimum="0" maximum="1" />
-                    <help-text >Stikkordene brukes av søket, men vises ikke på ssb.no. Komma brukes for å skille stikkord</help-text>
-                </input>
-            </items>
-        </field-set>
-    </form>
+	<display-name>
+		Statistikk
+	</display-name>
+	<super-type>
+		base:structured
+	</super-type>
+	<form>
+		<mixin name="statistic" />
+		<option-set name="showModifiedDate">
```

Comment:
Should this be wrapped in a field-set without a label just for consistency's sake? Like how you did for related articles.